spec.go

```diff
@@ -131,6 +131,10 @@ type Artifacts struct {
 	Directories *CreateArtifactDirectories `yaml:"createDirectories,omitempty" json:"createDirectories,omitempty"`
 	// ConfigFiles is a list of files that should be marked as config files in the package.
 	ConfigFiles map[string]ArtifactConfig `yaml:"configFiles,omitempty" json:"configFiles,omitempty"`
+	// DocFiles is a list of doc files included in the package
+	DocFiles []string `yaml:"docFiles,omitempty" json:"docFiles,omitempty"`
```

Comment:
The reason I like to make these structs is so that it is not a breaking change to add additional options.
I also think `subdir` is a thing relavent for these files (and licenses).

Reply:
Got it to work. I needed to handle the copy myself for the subpaths and emit the full path inside the %files section.